paddle/parameter/Parameter.cpp

```diff
@@ -375,10 +375,10 @@ bool Parameter::load(const std::string& filename) {
   std::ifstream fs(filename, std::ios_base::binary);
   if (!fs) {
     LOG(INFO) << "missing parameters [" << filename << "] while loading model.";
-    if (isStatic()) {
+    /*if (isStatic()) {
```

Comment:
It causes error when I tried to set static for some batch norm layers when their parameters are loaded from pre-trained model. I am trying to find out who added these lines and why.